utils/src/treev2/quickPickWizard/experiences/contextValueExperience.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../../index';
+import * as vscode from 'vscode';
+import { RecursiveQuickPickStep } from '../RecursiveQuickPickStep';
+import { getLastNode } from '../QuickPickWizardContext';
+import { NoResourceFoundError } from '../../../errors';
+import { AzureWizardPromptStep } from '../../../wizard/AzureWizardPromptStep';
+import { AzureWizard } from '../../../wizard/AzureWizard';
+import { isWrapper } from '../../../registerCommandWithTreeNodeUnwrapping';
+import { ResourceGroupsItem } from '../quickPickAzureResource/tempTypes';
+
+export async function contextValueExperience<TPick extends unknown>(context: types.IActionContext, tdp: vscode.TreeDataProvider<ResourceGroupsItem>, contextValueFilter: types.ContextValueFilter): Promise<TPick> {
+    const promptSteps: AzureWizardPromptStep<types.QuickPickWizardContext>[] = [
+        new RecursiveQuickPickStep(tdp, {
+            contextValueFilter: contextValueFilter,
+            skipIfOne: false,

```

Comment:
I feel like there are many times where we would want this to be true, such as only one sub, only one group, etc. It's only the last pick where this would be unwanted behavior, right?